app/models/LiabilityCalculation/Reliefs.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.LiabilityCalculation
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class Reliefs(
+                    residentialFinanceCosts: ResidentialFinanceCosts = ResidentialFinanceCosts(),
+                    reliefsClaimed: Seq[ReliefsClaimed] = Seq(ReliefsClaimed()),
+                    foreignTaxCreditRelief: ForeignTaxCreditRelief = ForeignTaxCreditRelief(),
+                    topSlicingRelief: TopSlicingRelief = TopSlicingRelief()
+                  )
+object Reliefs {
+  implicit val writes: OWrites[Reliefs] = Json.writes[Reliefs]
+  implicit val reads: Reads[Reliefs] = (
+    (JsPath \ "residentialFinanceCosts").read[ResidentialFinanceCosts] and
+      (JsPath \ "reliefsClaimed").read[Seq[ReliefsClaimed]] and
+      (JsPath \ "foreignTaxCreditRelief").read[ForeignTaxCreditRelief] and
+      (JsPath \ "topSlicingRelief").read[TopSlicingRelief]
+    ) (Reliefs.apply _)
+}
+
+case class ReliefsClaimed(`type`: Option[String] = None,
+                          amountUsed: Option[BigDecimal] = None)
+
+object ReliefsClaimed {
+  implicit val writes: OWrites[ReliefsClaimed] = Json.writes[ReliefsClaimed]
+  implicit val reads: Reads[ReliefsClaimed] = (
+    (JsPath \ "type").readNullable[String].map {
+      case Some("nonDeductableLoanInterest") => Some("nonDeductibleLoanInterest")
```

Comment:
Do we really want to change field reads from Deductable to Deductible? I think maintaining pure form of 1523 is better. But will leave it to you.

Reply:
I was only going by the comment in the spreadsheet - makes sense not to change the name imo